sdk-core/message.md

```diff
@@ -0,0 +1,26 @@
+## Message
+
+```
+dto Message:
+    data: bytes;
+    signature: bytes;
+```
+
+## MessageComputer
+
+```
+class MessageComputer:
```

Comment:
this class might also use a verify message signature or it doesn't belong here?

Reply:
Now, `verify()` is in sdk-wallet, but it would make sense somewhere in sdk-core, indeed (since only public key is used, therefore it should not necessarily be "in-the-wallet").